lib/perl/Genome/Qc/Config.pm

```diff
@@ -0,0 +1,36 @@
+package Genome::Qc::Config;
+
+use strict;
+use warnings;
+use Genome;
+
+class Genome::Qc::Config {
+    has => [
```

Comment:
I guess that doesn't quite work right if the result has the QC Config object itself as an input, but would if the name is the input.